3-fleetscope/modules/env_baseline/acm.tf

```diff
@@ -85,7 +85,9 @@ resource "google_project_iam_binding" "acm_wi_trace_agent" {
   role = "roles/cloudtrace.agent"
   members = [
     "principal://iam.googleapis.com/projects/${data.google_project.cluster_project.number}/locations/global/workloadIdentityPools/${var.fleet_project_id}.svc.id.goog/subject/ns/config-management-monitoring/sa/default",
-    "principal://iam.googleapis.com/projects/${data.google_project.cluster_project.number}/locations/global/workloadIdentityPools/${var.fleet_project_id}.svc.id.goog/subject/ns/default/sa/cymbal-bank", #TODO rename/move
```

Comment:
The changes not contemplated on the linked commit are on the integrations tests - currently the appinfra tests (`appinfra_test.go`) only tests the `shared` environment. 

`development`, `nonproduction` and `production` would also need to be tested as the permission assignments are specific to these. An alternative to this would be to assign the permissions on the `shared` environment.

Reply:
Having the permissions assignment on 5-appinfra will give people with access to the application infrastructure permission to edit IAM policies in the cluster/fleet project, which could lead to privilege escalation